test/sets.jl

```diff
@@ -1037,6 +1037,8 @@ end
     @test !isempty(A)
     A = empty!(A)
     @test isempty(A)
+    @test isnothing(sizehint!(A, 10))
```

Comment:
Is there a better way to test this `sizehint!()` call? Maybe by looking at the length of `.ht` before and after the `sizehint!()` call?

Reply:
Maybe add a new `TODO`?

https://github.com/JuliaLang/julia/blob/10f294b422ce56759639e99399aec5c407bafc00/base/idset.jl#L81-L93

Reply:
Ah, I didn't realize it was currently a no-op.